BuildAssets/repos/test-rpm-dists.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+if [[ -z "$PUBLISH_LINUX_EXTERNAL" || -z "$OCTOPUS_CLI_SERVER" || -z "$OCTOPUS_CLI_API_KEY" \
```

Comment:
as above re one line per var

Reply:
Okay done, but with a joint explanation for the Octopus variables as they all relate.